### DISCUSSION TITLE
How do I add logo/icon to tokens minted on GoChain?

### DISCUSSION THREAD

**Discussion by SimonSee** [gochain-io/docs#113] [General]
Ive been learning how to mint tokens on GoChain but can figure out how to add a logo/icon image. Is it possible?

**Comment by treeder** [comment]
Where do you want the logo to show up?  Logos aren't in smart contracts, always a separate thing.

**Comment by SimonSee** [comment]
Hello,

When I say logos I mean having a icon that shows up on exchanges, next to
the symbol. With etherium you can add them via etherscan, however there
doesn't seem to be an option on the gochain explorer.

I'm new to gochain, discovered it via the Earth2.io rumors so have been
minting a few tokens on Gochain with the idea of having a token for use in
a city I've got in the E2 metaverse.

Thanks for your help.

Simon.

**Reply by treeder** [reply]
That's up to the exchanges, not really related to what's on chain. You'll have to provide them a logo if you get listed. 

A lot of wallets/exchanges use this repository for logos: https://github.com/trustwallet/assets

**Comment by SimonSee** [comment]
Ah, thanks a million. Will Earth2 be using the GoChain blockchain? I'm
loving learning about GoChain but am doing so to build community currencies
for Earth2 guilds. It would be a shame if it was all in vain.

Thanks again for your help.

Simon.
